library/src/main/java/com/novoda/downloadmanager/lib/RequestBatch.java

```diff
@@ -0,0 +1,50 @@
+package com.novoda.downloadmanager.lib;
+
+import android.content.ContentValues;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RequestBatch {
+
+    private final BatchInfo batchInfo;
+    private final List<Request> requests;
+
+    public static RequestBatch newInstance(String title, String description, String bigPictureUrl) {
```

Comment:
*Personally* I would not go for `newInstance` because I think that is a very Novoda-y thing and not a general pattern - external users wouldn't be very familiar with it. Perhaps a builder would be nicer? I feel like the params should be optional too